pkg/api/docs/definitions/dashboard_versions.go

```diff
@@ -38,6 +38,7 @@ import "github.com/grafana/grafana/pkg/models"
 
 // swagger:parameters getDashboardVersions getDashboardVersion restoreDashboardVersion
 // swagger:parameters getDashboardPermissions postDashboardPermissions
+// swagger:parameters rendeReportPDF
```

Comment:
A typo probably?

Reply:
indeed, thanks! but I committed this file by mistake here: it suits better on the original PR so I'll move it there.